README.md

```diff
@@ -162,3 +162,8 @@ environment. It will refuse to start if anything is badly configured.
 
 The checks can be bypassed with the environment variable
 `ODOO_CLOUD_PLATFORM_UNSAFE` set to `1`.
+
+### Attachment storage inactivation 
+
+To prevent object storage to be accessed while failing for any kind of reason
+set this environment variable `ATTACHMENT_STORAGE_INACTIVE` set to `1`.
```

Comment:
```suggestion
set this environment variable `DISABLE_ATTACHMENT_STORAGE` set to `1`.
```

Reply:
Pushed ^^

Reply:
Maybe GitHub is tricking me, but now it's `ATTACHMENT_STORAGE_INACTIVE`